Expensify/App
# Font-family difference between platforms

### Details

This PR fixes the font-family difference between platform

### Fixed Issues

https://github.com/Expensify/Expensify.cash/issues/2491

### Tests

1. Send the following message on mobile
```
  *`test`*
  _`test`_
```
2. Italics and bold must be applied.
3. View the message on desktop / web / ios / android

### QA Steps



1. Click on the text input
2. Send a message the following messages
```
1. *`bold`*
2. _`italic`_
3. `normal`
4. *_`italic and bold`_*
```
3. The fonts should be applied like in web / desktop

### Tested On


### Screenshots


#### Web
![Screen Shot 2021-04-30 at 00 24 25](https://user-images.githubusercontent.com/14885967/116645571-86da6380-a94c-11eb-8149-f60d87b0bed7.png)

#### Mobile Web

![Screen Shot 2021-04-30 at 00 58 23](https://user-images.githubusercontent.com/14885967/116646754-40d2cf00-a94f-11eb-86fe-ae0ea553d965.png)

#### Desktop
![Screen Shot 2021-04-30 at 00 26 52](https://user-images.githubusercontent.com/14885967/116645626-96f24300-a94c-11eb-98b9-59e73c326a17.png)

#### iOS
![Simulator Screen Shot - iPhone 11 - 2021-04-30 at 00 17 39](https://user-images.githubusercontent.com/14885967/116645639-9c4f8d80-a94c-11eb-89a4-a3d937df58b9.png)

#### Android
![WhatsApp Image 2021-04-30 at 00 37 04](https://user-images.githubusercontent.com/14885967/116645642-9fe31480-a94c-11eb-9f2b-b3b4068ae271.jpeg)
